platform/vendor/vendor-helm-chart.sh

```diff
@@ -46,6 +46,8 @@ fi
 echo -e "${C_GREEN}Vendoring ${opt_c}...${C_RESET_ALL}"
 
 # helm pull to tmp folder
+basedir="$(dirname "${opt_d}")"
+[ -d "${basedir}" ] || mkdir -p "${basedir}"
```

Comment:
Nitpicky, but if you use `mkdir -p` the test before becomes useless.

Reply:
I'm being overly paranoid never knowing if some `mkdir` implementation would fail if the directory already exists, but maybe all `mkdir` commands handle that right.

Reply:
🤞